app/views/users/settings.html.erb

```diff
@@ -1,6 +1,10 @@
 <% content_for :title, t('students.account_settings') %>
-<% if @current_user.student? %>
-  <%= form_with url: 'update_settings', method: 'patch', model: @current_user, local: true do |f| %>
+<%= form_with url: 'update_settings', method: 'patch', model: User, local: true do |f| %>
+  <p>
+    <%= f.label :locale, t(:language) %>
```

Comment:
You can check for other uses of the `:language` string, but now we should move it to `models/users/en.yml` under `attributes` as well.